src/form-utils/form-utils.ts

```diff
@@ -0,0 +1,170 @@
+import i18n from 'i18next';
```

Comment:
`i18n` will need to be initialized after the webpack merge is merged (or you can cherrypick the latest commit) :)

Reply:
This one i would make afer merge both PRs, and use your util.